third_party/licenses/licenses.csv

```diff
@@ -0,0 +1,701 @@
+# Generated by https://github.com/google/go-licenses/v2. DO NOT EDIT.
+cloud.google.com/go, https://github.com/googleapis/google-cloud-go/blob/v0.51.0/LICENSE, Apache-2.0
+cloud.google.com/go, https://github.com/googleapis/google-cloud-go/blob/v0.51.0/cmd/go-cloud-debug-agent/internal/debug/elf/elf.go, BSD-2-Clause
+github.com/Azure/go-autorest/autorest, https://github.com/Azure/go-autorest/blob/v0.9.6/LICENSE, Apache-2.0
+github.com/Azure/go-autorest/autorest/adal, https://github.com/Azure/go-autorest/blob/v0.8.2/LICENSE, Apache-2.0
+github.com/Azure/go-autorest/autorest/date, https://github.com/Azure/go-autorest/blob/v0.2.0/LICENSE, Apache-2.0
+github.com/Azure/go-autorest/logger, https://github.com/Azure/go-autorest/blob/v0.1.0/LICENSE, Apache-2.0
+github.com/Azure/go-autorest/tracing, https://github.com/Azure/go-autorest/blob/v0.5.0/LICENSE, Apache-2.0
+github.com/beorn7/perks, https://github.com/beorn7/perks/blob/v1.0.1/LICENSE, MIT
+github.com/beorn7/perks, https://github.com/beorn7/perks/blob/v1.0.1/README.md, MIT
+github.com/cespare/xxhash/v2, https://github.com/cespare/xxhash/blob/v2.1.1/LICENSE.txt, MIT
+github.com/coreos/go-semver, https://github.com/coreos/go-semver/blob/v0.3.0/LICENSE, Apache-2.0
+github.com/coreos/go-systemd/v22, https://github.com/coreos/go-systemd/blob/v22.3.2/LICENSE, Apache-2.0
+github.com/davecgh/go-spew, https://github.com/davecgh/go-spew/blob/v1.1.1/LICENSE, ISC
```

Comment:
There are too many entries for go-spew, gogo/protobuf, ...
They are false positives detected by the tool, because these repos have a license header on every file.

In these cases, you should override for these modules in `go-licenses.yaml`. I believe a lot of libraries you use are also KFP dependencies, so you can use what I prepare for KFP as an easy reference: https://github.com/kubeflow/pipelines/blob/master/v2/go-licenses.yaml.

Reply:
fix